Header header

# дистанция назад, [м], [0, inf)
float32 distance_backward

# дистанция вперед, [м], [0, inf)
float32 distance_forward

# дистанция влево, [м], [0, inf)
float32 distance_leftward

# дистанция вправо, [м], [0, inf)
float32 distance_rightward